baseline/OSPS-VM.yaml

```
@@ -297,3 +164,144 @@ controls:
           To the degree possible, this information should include affected
           version(s), how a consumer can determine if they are vulnerable, and
           instructions for mitigation or remediation.
+      - id: OSPS-VM-04.02
+        text: |
+          The project's released software assets MUST include VEX data to
+          provide information about the exploitability of vulnerabilities.
+        applicability:
+          - Maturity Level 3
+        recommendation: |
+          Include a VEX file or VEX data in an SBOM as part of the project's 
+          released software assets. This data should provide information about
+          the exploitability of vulnerabilities, including which conditions or
+          configurations are necessary for the vulnerability to be exploited
+          or non-applicable.
```

Discourage VEX in SBOMs. While it is perfectly acceptable to use the SBOM formats to communicate VEX, we want to keep the VEX data separate from the bill of materials

```suggestion
          Establish a VEX feed communicating the exploitability status of 
          known vulnerabilities, including assessment details or any
          mitigations in place preventing vulnerable code from being
          executed.
```